accounts-db/src/ancient_append_vecs.rs

```diff
@@ -577,6 +624,14 @@ impl AccountsDb {
             })
             .count()
             .saturating_sub(randoms as usize);
+        // ideal storage size is total alive bytes of ancient storages / half of max ancient slots
+        tuning.ideal_storage_size = NonZeroU64::new(
+            (infos.total_alive_bytes.0 / tuning.max_ancient_slots.max(1) as u64 * 2).max(5_000_000),
```

Comment:
i think ideal storage size needs to be dynamically calculated. Here, I picked max ancient slots / 2 should be able to hold all of the alive bytes. We could be more aggressive and maybe say / 4. This constant can be true no matter the width. 10k is the default width today. @HaoranYi this is a new pr.